### DISCUSSION TITLE
Trapezoid with a given thickness

### DISCUSSION THREAD

**Discussion by MarcelSchaible** [BelfrySCAD/BOSL2#1417] [Q&A]
Is it possible to create a trapezoid with a user defined thickness?

**Comment by revarbat** [comment]
I'm not quite sure what you mean by thickness in this context.

**Reply by MarcelSchaible** [reply]
I used probably the wrong term. I want to design a part which has the shape of a trapezoid, but has some depth in 3D.
![x](https://github.com/BelfrySCAD/BOSL2/assets/1245232/d9bcc4a6-4b6f-455c-8cb6-6ed9114449fa)

**Reply by revarbat** [reply]
That looks like a `cube()` with a `prismoid()` attached to the top of it.

```openscad
cube([60,20,10], anchor=BOT)
  attach(TOP, BOT, overlap=0.01)
    prismoid([60,20], [30,20], h=30.01);
```
